bot/modules/community/scenes.communityAdmin.js

```diff
@@ -0,0 +1,37 @@
+const { Scenes } = require('telegraf');
+
+const CommunityEvents = require('../events/community');
+
+module.exports = () => {
+  const scene = new Scenes.WizardScene('COMMUNITY_ADMIN', async ctx => {
+    const { community } = ctx.scene.state;
```

Comment:
`community.nostr_public_key`  should have an empty value `''` to avoid show `undefined` in case users don't have this value setted up